FLEDGE.md

```diff
@@ -803,3 +811,184 @@ These signals were requested in [issue 435](https://github.com/WICG/turtledove/i
 
 This could be handled by a `reportLoss()` function running in the worklet.  Alternatively, the model of [SPURFOWL](https://github.com/AdRoll/privacy/blob/main/SPURFOWL.md) (an append-only datastore and later aggregate log processing) could be a good fit for this use case.  The details here are yet to be determined.
+
+### 6. Additional Bids
+
+In addition to bids generated by interest groups, sellers can enable buyers to introduce bids generated outside of the auction. We refer to these as additional bids. Additional bids are commonly triggered using contextual signals. By having more contextual bids participate in the auction, we're reducing the leakage that's learned by detecting whether or not an auction produced an interest-group-based winning ad.
+
+Buyers compute the additional bids, likely as part of a contextual auction. Buyers need to package up each additional bid using a new data structure that encapsulates all of the information needed for the additional bid to compete against other bids in a Protected Audience auction. Sellers are responsible for passing additional bids to the browser at Protected Audience auction time
+
+Each additional bid is expressed using the following JSON data structure:
+
+```
+const additionalBid = {
+  "bid": {
+    // Fields analogous to those returned by generateBid()
+    "ad": 'ad-metadata',
+    "adCost": 2.99,
+    "bid": 1.99,
+    "bidCurrency": "USD",
+    "render": "https://www.example-dsp.com/ad/123.jpg",
+    "adComponents": [adComponent1, adComponent2],
+    "allowComponentAuction": true,
+    "modelingSignals": 123,
+  },
+  // These facilitate running reportWin() if this bid wins the auction.
+  "interestGroup": {
+    // These are be passed to reportWin()
+    "owner": "https://www.example-dsp.com"
+    "name": "campaign123",
+    // This is used for its definition of reportWin()
+    "biddingLogicURL": "https://www.example-dsp.com/bid_logic.js"
+  },
+  // Negative interest groups are described in more detail below.
+  // Only one of negativeInterestGroup or negativeInterestGroups
+  // should be specified on a given bid.
+  "negativeInterestGroup": "campaign123_negative_interest_group",
+  "negativeInterestGroups": {
+    joiningOrigin: "https://www.example-advertiser.com",
+    interestGroupNames: [
+      "example_advertiser_negative_interest_group_a",
+      "example_advertiser_negative_interest_group_b",
+    ]
+  },
+
+  // The following fields are used to prevent replay of this bid.
+  // auctionNonce is described in greater detail below.
+  "auctionNonce": "12345678-90ab-cdef-fedcba09876543210",
+  // seller and topLevelSeller must match the auction's configuration.
+  "seller": "https://www.example-ssp.com",
+  "topLevelSeller": "https://www.another-ssp.com"
+}
```

Comment:
Buyers have so far remained insulated from multi-seller mechanics/identities.  If I read this correctly, a buyer's signed additional bid must contain the topLevelSeller.

Reply:
The `seller` and `topLevelSeller` fields here are meant to echo those present in the `browserSignals` argument to `generateBid()`. I've removed the brief descriptions of these fields inlined in the additional bid JSON snippet above, and instead add richer descriptions in the paragraphs below it, including a clarification of this point.

Reply:
A `seller` may have a relationship with a `topLevelSeller` that is opaque to a buyer. Or a buyer might not care who exactly the `topLevelSeller` is. Is there a mechanism for a buyer to specify an additional bid in such a way that matches any `topLevelSeller`, including no `topLevelSeller` at all?